bounce/server/api/clubs.py

```diff
@@ -25,6 +25,18 @@ class ClubEndpoint(Endpoint):
             raise APIError('No such club', status=404)
         return response.json(club_data, status=200)
 
+    #@validate(None, GetClubResponse)
+    # Although this method uses a GET request, I don't know if we can
+    # validate the search function if it gives a list
+    # of SQL objects ... should we create another method for validation?
+    async def search(self, _, user_input):
+        """Handles a full text search by returning clubs with content
+        that include lexemes from the user input."""
+        queried_clubs = club.search_clubs(self.server.db_session, user_input)
+        if queried_clubs:
+            for c in queried_clubs:
+                print(dict(c))
```

Comment:
You never return an actual response here....

Reply:
You also have a `lint` error here: https://travis-ci.org/ubclaunchpad/bounce/builds/413874713#L640.

Reply:
ok.  Looks like pylint doesn't like the "c" variable name.  I'll change it to club.